src/translations/en.ts

```diff
@@ -222,6 +222,7 @@ export const en = {
     lnFilter: 'Logical Node Filter',
     missing: 'No IED',
     toggleChildElements: 'Toggle child elements',
+    settings: 'Show Services the IED/port provides',
```

Comment:
Rather than `port` should we match Jakob's translation and retain the SCL-specific name with:

```suggestion
    settings: 'Show Services the IED/AccessPoint provides',
```